karma-ci-cov.conf.js

```diff
@@ -6,6 +6,10 @@ module.exports = function (config) {
 			"src/**/*.ts": ["ui5-transpile"],
 			"test/**/*.ts": ["ui5-transpile"]
 		},
+		proxies: {
```

Comment:
This should maybe better tackled by the `karma-ui5` plugin - but for now it is the best solution to get this fixed. This issue comes with flat libraries!